clientd3d/dialog.h

```diff
@@ -55,6 +55,7 @@ typedef struct {
    char        *description;
    char        *fixed_string;
    char        *url;
+   int          rarity;      /* Item rarity grade */
```

Comment:
Shouldn't the type of this be item_rarity_grade instead of int?

Reply:
Yes, correct.  I changed the type from int to item_rarity_grade.

This was a learning experience for me.  I figured since enums are implemented as integers for practical purposes, this was fine.

I did some research after your comment.  I now understand that enums are distinct types from integers for type-checking purposes, even though they are implemented as integers.